kegeland-frontend/src/components/DataTable.tsx

```diff
@@ -32,13 +32,14 @@ import {
   getSortedRowModel,
   useReactTable,
 } from '@tanstack/react-table';
+import { useNavigate } from 'react-router-dom';
 
 export type DataTableProps<T extends object> = {
   data: T[];
   columns: ColumnDef<T, any>[];
 };
 
-const DataTable = <T extends object>({ data, columns }: DataTableProps<T>) => {
+const DataTable = <T extends { object?: any, id?: string } >({ data, columns }: DataTableProps<T>) => {

```

Comment:
Why do you use type any here? Could you make a custom type including the types that should be accepted?

Reply:
Could be removed